script.js

```diff
@@ -106,20 +109,23 @@ function pause(forResetOrToggle = false) {
 function startStopWatch() {
     
     // For checking if the timer is already running.
-    hasPausedStopWatch = false;
-    
-    // Starting time in milliseconds.
-    let startTime = Date.now() - elapsedTime;
-    
-    timerIntervalStopWatchId = setInterval(function() {
-        
-        // Calculating elapsed time in milliseconds.
-        elapsedTime = Date.now() - startTime;
+    if(stopwatchStarted == false){
```

Comment:
Instead of using the conditionally equal-to `==` operator, why don't you use the logical not `!` operator *( as you doing some logical operation )*?

This variable will not be needed if you use the existing `hasPausedStopWatch` variable.